src/main/kotlin/br/com/zup/op/events/producer/KafkaProducerConfig.kt

```diff
@@ -12,6 +15,8 @@ import java.util.*
 @Configuration
 class KafkaProducerConfig {
```

Comment:
Rename `KafkaProducerConfig` to `KafkaConfig`

Reply:
The class will be rename after @Willcd  commit a code publish event